### DISCUSSION TITLE
Now I use bendable lines, but when I connect, if they are at the same point, the two lines merge, how can I make them not merge?

### DISCUSSION THREAD

**Discussion by xtt-nora** [maxGraph/maxGraph#479] [Q&A]
through I use setPortsEnabled equal true ，I use different point in cell， two lines edges  are merged 。, how can I make them not merge?

**Comment by tbouffard** [comment]
Hi can you provide a screenshot or a video of what you see?
Can you also provide the code your are using?

This will help to better understand what is going on.
Thanks in advance

**Comment by tbouffard** [comment]
This may be a similar request to #371 or #533
